axelrod/tests/unit/test_fingerprint.py

```diff
@@ -119,6 +233,27 @@ def test_fingerprint_with_filename(self):
             data = out.read()
             self.assertEqual(len(data.split("\n")), 10)
 
+    @unittest.skipIf(axl.on_windows,
```

Comment:
new tests.  hopefully these work.  submitting blind as on windows machine.  off to work. fix later if needed.